library/src/config.rs

```diff
@@ -1,4 +1,5 @@
 // This file handles the global config for the updater library.
+use anyhow::bail;
```

Comment:
I don't bother to import things when I'm only calling them once. 🤷

Reply:
Fixed.